.github/workflows/publish-on-merge.yaml

```diff
@@ -0,0 +1,34 @@
+name: Publish on Release PR Merge
+
+on:
+  push:
+    branches: [master]
+    paths: ['packages/*/package.json']
+
+jobs:
+  publish:
+    if: contains(github.event.head_commit.message, 'chore(release): version bump')
+    runs-on: ubuntu-latest
+    environment: production
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install dependencies
+        run: yarn install --immutable
+
+      - name: Build packages
+        run: yarn build
+
+      - name: Configure NPM
+        run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> .npmrc
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_PUBLISH_TOKEN }}
+
+      - name: Publish to NPM and create tags
```

Comment:
I don't think the tags will be pushed because it normally happens on `lerna version` and now it's disabled `"gitTagVersion": false`.

So we have to tag manually here

Reply:
hmm true, maybe we turn on gitTagVersion again? i think the risk review process might not prevent us from tagging, and the tagging seems nice